llama_stack/cli/model/list.py

```diff
@@ -5,12 +5,52 @@
 # the root directory of this source tree.
 
 import argparse
+import os
+import time
 
 from llama_stack.cli.subcommand import Subcommand
 from llama_stack.cli.table import print_table
+from llama_stack.distribution.utils.config_dirs import DEFAULT_CHECKPOINT_DIR
 from llama_stack.models.llama.sku_list import all_registered_models
 
 
+def _convert_size_to_gb(size):
+    return size / (1024**3)
+
+
+def _get_model_size(model_dir):
+    model_size = 0
+    for dirpath, _, files in os.walk(model_dir):
+        for file in files:
+            file_path = os.path.join(dirpath, file)
+            model_size += os.path.getsize(file_path)
+    return model_size
+
+
+def _run_model_list_downloaded_cmd() -> None:
+    headers = ["Model", "Size", "Modified Time"]
+
+    rows = []
+    for model in os.listdir(DEFAULT_CHECKPOINT_DIR):
+        abs_path = os.path.join(DEFAULT_CHECKPOINT_DIR, model)
+        space_usage = _get_model_size(abs_path)
+        model_size = f"{_convert_size_to_gb(space_usage):.2f} GB"
```

Comment:
nit: remove _convert_size_to_gb? 
```suggestion
        model_size = f"{space_usage / (1024**3):.2f} GB"
```